com.hahoto.onelimx.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.hahoto.onelimx
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: 1LimX
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+modules:
+  - name: 1LimX
+    # There are other types of buildsystems, like autotools, cmake, cmake-ninja, simple, qmake
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    sources:
+      - type: file
+        url: https://www.dropbox.com/scl/fi/fli2wzrhyslc2vs8hnpfp/1LimX-Linux-v1.2.10-amd64-setup.deb?rlkey=fnhru84c89qm6fq8erjdox9v4&dl=1
```

Comment:
This is not a credible source to host application code or binary or whatever you are hosting.